test_resource/fake_response/fake_sign_in_response.json

```diff
@@ -0,0 +1,13 @@
+{
+  "data": {
+    "id": 10,
+    "type": "token",
+    "attributes": {
+      "access_token": "lbxD2K2BjbYtNzz8xjvh2FvSKx838KBCf79q773kq2c",
```

Comment:
better not leaving the actual token here - understanding this is non-production app but it is better for you to learn this at the same time.

`"access_token": "a fake token",` should be enough